<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1460</width>
    <height>818</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Movimenti</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QFrame" name="frame_3">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>251</width>
      <height>851</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border:none;
background-color:rgb(20, 18, 29)</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QPushButton" name="loanButton">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>150</y>
       <width>191</width>
       <height>51</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton 
{
border-radius:15px;
background-color: rgb(51, 49, 77);
color:rgb(255, 255, 255);
}
QPushButton::pressed
{
	background-color:rgb(30, 97, 255)
}</string>
     </property>
     <property name="text">
      <string>Nuovo Prestito</string>
     </property>
     <property name="icon">
      <iconset>
       <normaloff>../img/Icona Prestito.png</normaloff>../img/Icona Prestito.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>40</width>
       <height>40</height>
      </size>
     </property>
    </widget>
    <widget class="QPushButton" name="consultationButton">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>230</y>
       <width>191</width>
       <height>51</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton 
{
border-radius:15px;
background-color: rgb(51, 49, 77);
color:rgb(255, 255, 255);
}
QPushButton::pressed
{
	background-color:rgb(30, 97, 255)
}</string>
     </property>
     <property name="text">
      <string>Nuova Consultazione</string>
     </property>
     <property name="icon">
      <iconset>
       <normaloff>../img/Icona Consultazione.png</normaloff>../img/Icona Consultazione.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>40</width>
       <height>40</height>
      </size>
     </property>
    </widget>
    <widget class="QPushButton" name="backButton">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>720</y>
       <width>81</width>
       <height>51</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton 
{
	border-radius:16px;
	background-color: rgb(24, 22, 29);
}
QPushButton::pressed
{
	background-color:rgb(95, 95, 95)
}</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset>
       <normaloff>../img/go back.png</normaloff>../img/go back.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>40</width>
       <height>40</height>
      </size>
     </property>
    </widget>
    <widget class="QPushButton" name="infoButton">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>310</y>
       <width>191</width>
       <height>51</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton 
{
border-radius:15px;
background-color: rgb(51, 49, 77);
color:rgb(255, 255, 255);
}
QPushButton::pressed
{
	background-color:rgb(30, 97, 255)
}</string>
     </property>
     <property name="text">
      <string>Scheda Movimento</string>
     </property>
     <property name="icon">
      <iconset>
       <normaloff>../img/Icona Information.png</normaloff>../img/Icona Information.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>40</width>
       <height>40</height>
      </size>
     </property>
    </widget>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>30</y>
       <width>91</width>
       <height>81</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="textFormat">
      <enum>Qt::RichText</enum>
     </property>
     <property name="pixmap">
      <pixmap>../img/Icona Movimenti.png</pixmap>
     </property>
     <property name="scaledContents">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QPushButton" name="givingbookButton">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>390</y>
       <width>191</width>
       <height>51</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton 
{
border-radius:15px;
background-color: rgb(51, 49, 77);
color:rgb(255, 255, 255);
}
QPushButton::pressed
{
	background-color:rgb(30, 97, 255)
}</string>
     </property>
     <property name="text">
      <string>Rientro Prestito</string>
     </property>
     <property name="icon">
      <iconset>
       <normaloff>../img/book_back.png</normaloff>../img/book_back.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>32</width>
       <height>32</height>
      </size>
     </property>
    </widget>
    <widget class="QPushButton" name="deleteMovements">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>470</y>
       <width>191</width>
       <height>51</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton 
{
border-radius:15px;
background-color: rgb(51, 49, 77);
color:rgb(255, 255, 255);
}
QPushButton::pressed
{
	background-color:rgb(30, 97, 255)
}</string>
     </property>
     <property name="text">
      <string>Elimina Movimento</string>
     </property>
     <property name="icon">
      <iconset>
       <normaloff>../img/delete.png</normaloff>../img/delete.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>40</width>
       <height>40</height>
      </size>
     </property>
    </widget>
   </widget>
   <widget class="QFrame" name="frame">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>50</y>
      <width>1101</width>
      <height>161</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-radius: 20px;
border-style: outset;
border-width: 2px;
border-color:rgb(27,25,39);
background-color:rgba(0, 192, 225, 125)</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QLineEdit" name="searchField">
     <property name="geometry">
      <rect>
       <x>290</x>
       <y>40</y>
       <width>511</width>
       <height>41</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">border:none;
background-color: rgba(23, 28, 78, 17);
color:rgb(255, 255, 255);
font: 14pt &quot;Source Code Pro for Powerline&quot;;
border:none;
border-bottom: 1px solid rgba(255, 255, 255, 100);

padding: 10px</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="frame">
      <bool>true</bool>
     </property>
     <property name="cursorPosition">
      <number>0</number>
     </property>
     <property name="dragEnabled">
      <bool>false</bool>
     </property>
     <property name="placeholderText">
      <string>Cerca Movimento</string>
     </property>
     <property name="clearButtonEnabled">
      <bool>false</bool>
     </property>
    </widget>
    <widget class="QComboBox" name="searchMode">
     <property name="geometry">
      <rect>
       <x>670</x>
       <y>100</y>
       <width>131</width>
       <height>41</height>
      </rect>
     </property>
     <property name="palette">
      <palette>
       <active>
        <colorrole role="Button">
         <brush brushstyle="SolidPattern">
          <color alpha="125">
           <red>0</red>
           <green>192</green>
           <blue>225</blue>
          </color>
         </brush>
        </colorrole>
        <colorrole role="Base">
         <brush brushstyle="SolidPattern">
          <color alpha="125">
           <red>0</red>
           <green>192</green>
           <blue>225</blue>
          </color>
         </brush>
        </colorrole>
        <colorrole role="Window">
         <brush brushstyle="SolidPattern">
          <color alpha="125">
           <red>0</red>
           <green>192</green>
           <blue>225</blue>
          </color>
         </brush>
        </colorrole>
       </active>
       <inactive>
        <colorrole role="Button">
         <brush brushstyle="SolidPattern">
          <color alpha="125">
           <red>0</red>
           <green>192</green>
           <blue>225</blue>
          </color>
         </brush>
        </colorrole>
        <colorrole role="Base">
         <brush brushstyle="SolidPattern">
          <color alpha="125">
           <red>0</red>
           <green>192</green>
           <blue>225</blue>
          </color>
         </brush>
        </colorrole>
        <colorrole role="Window">
         <brush brushstyle="SolidPattern">
          <color alpha="125">
           <red>0</red>
           <green>192</green>
           <blue>225</blue>
          </color>
         </brush>
        </colorrole>
       </inactive>
       <disabled>
        <colorrole role="Button">
         <brush brushstyle="SolidPattern">
          <color alpha="125">
           <red>0</red>
           <green>192</green>
           <blue>225</blue>
          </color>
         </brush>
        </colorrole>
        <colorrole role="Base">
         <brush brushstyle="SolidPattern">
          <color alpha="125">
           <red>0</red>
           <green>192</green>
           <blue>225</blue>
          </color>
         </brush>
        </colorrole>
        <colorrole role="Window">
         <brush brushstyle="SolidPattern">
          <color alpha="125">
           <red>0</red>
           <green>192</green>
           <blue>225</blue>
          </color>
         </brush>
        </colorrole>
       </disabled>
      </palette>
     </property>
     <property name="styleSheet">
      <string notr="true">QComboBox{
	border: none;
}

QAbstractView{
	background-color: rgb(3, 158, 182);
}

QWidget{
	border-radius: 0px;
}

</string>
     </property>
     <property name="currentIndex">
      <number>0</number>
     </property>
     <property name="maxVisibleItems">
      <number>7</number>
     </property>
     <property name="frame">
      <bool>true</bool>
     </property>
     <item>
      <property name="text">
       <string>In tutti i campi</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Titolo</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Utente</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Data</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>ISBN</string>
      </property>
     </item>
    </widget>
    <widget class="QRadioButton" name="loanRadio">
     <property name="geometry">
      <rect>
       <x>300</x>
       <y>110</y>
       <width>71</width>
       <height>20</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">border:none;
background-color:transparent;</string>
     </property>
     <property name="text">
      <string>Prestiti</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="consultationRadio">
     <property name="geometry">
      <rect>
       <x>390</x>
       <y>110</y>
       <width>111</width>
       <height>20</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">border:none;
background-color:transparent;</string>
     </property>
     <property name="text">
      <string>Consultazioni</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="backRadio">
     <property name="geometry">
      <rect>
       <x>510</x>
       <y>110</y>
       <width>111</width>
       <height>20</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">border:none;
background-color:transparent;</string>
     </property>
     <property name="text">
      <string>Rientri</string>
     </property>
    </widget>
   </widget>
   <widget class="QTableWidget" name="movementTable">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>240</y>
      <width>1111</width>
      <height>541</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
      <horstretch>40</horstretch>
      <verstretch>40</verstretch>
     </sizepolicy>
    </property>
    <property name="minimumSize">
     <size>
      <width>100</width>
      <height>200</height>
     </size>
    </property>
    <property name="sizeIncrement">
     <size>
      <width>0</width>
      <height>0</height>
     </size>
    </property>
    <property name="mouseTracking">
     <bool>false</bool>
    </property>
    <property name="tabletTracking">
     <bool>false</bool>
    </property>
    <property name="accessibleName">
     <string/>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">
QWidget{
	background-color: rgb(27, 25, 39);
font: 14pt &quot;Source Code Pro for Powerline&quot;;
background-color:qlineargradient(spread:reflect, x1:0, y1:0, x2:1, y2:0, stop:0 rgba(56, 54, 65, 255), stop:1 rgba(56, 54, 65, 130));
qproperty-alignment: 'AlignLeft';
background-color: rgb(16, 14, 23,220);
alternate-background-color: rgb(38, 35, 63);
color: white;
border:none;
border-radius:10px;
}

QHeaderView::section{
	font: 14pt bold;
	color:rgb(121, 212, 226);
	background-color:  rgba(11, 11, 13, 50);
	border-width: 5px;

}</string>
    </property>
    <property name="lineWidth">
     <number>1</number>
    </property>
    <property name="verticalScrollBarPolicy">
     <enum>Qt::ScrollBarAsNeeded</enum>
    </property>
    <property name="horizontalScrollBarPolicy">
     <enum>Qt::ScrollBarAlwaysOff</enum>
    </property>
    <property name="dragDropOverwriteMode">
     <bool>true</bool>
    </property>
    <property name="alternatingRowColors">
     <bool>true</bool>
    </property>
    <property name="selectionBehavior">
     <enum>QAbstractItemView::SelectRows</enum>
    </property>
    <property name="showGrid">
     <bool>false</bool>
    </property>
    <property name="gridStyle">
     <enum>Qt::SolidLine</enum>
    </property>
    <property name="sortingEnabled">
     <bool>true</bool>
    </property>
    <property name="rowCount">
     <number>4</number>
    </property>
    <attribute name="horizontalHeaderVisible">
     <bool>true</bool>
    </attribute>
    <attribute name="horizontalHeaderCascadingSectionResizes">
     <bool>false</bool>
    </attribute>
    <attribute name="horizontalHeaderMinimumSectionSize">
     <number>50</number>
    </attribute>
    <attribute name="horizontalHeaderDefaultSectionSize">
     <number>180</number>
    </attribute>
    <attribute name="horizontalHeaderHighlightSections">
     <bool>false</bool>
    </attribute>
    <attribute name="horizontalHeaderShowSortIndicator" stdset="0">
     <bool>true</bool>
    </attribute>
    <attribute name="horizontalHeaderStretchLastSection">
     <bool>true</bool>
    </attribute>
    <attribute name="verticalHeaderVisible">
     <bool>false</bool>
    </attribute>
    <attribute name="verticalHeaderCascadingSectionResizes">
     <bool>false</bool>
    </attribute>
    <attribute name="verticalHeaderHighlightSections">
     <bool>true</bool>
    </attribute>
    <attribute name="verticalHeaderShowSortIndicator" stdset="0">
     <bool>false</bool>
    </attribute>
    <attribute name="verticalHeaderStretchLastSection">
     <bool>false</bool>
    </attribute>
    <row>
     <property name="text">
      <string>New Row</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>New Row</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>New Row</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>New Row</string>
     </property>
    </row>
    <column>
     <property name="text">
      <string>Data Mov.</string>
     </property>
     <property name="font">
      <font>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
    </column>
    <column>
     <property name="text">
      <string>ISBN</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Titolo Doc.</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Codice Fiscale</string>
     </property>
     <property name="font">
      <font>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Utente</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Recapito</string>
     </property>
     <property name="font">
      <font>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
    </column>
    <item row="0" column="0">
     <property name="text">
      <string/>
     </property>
     <property name="flags">
      <set>ItemIsEnabled</set>
     </property>
    </item>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>20</y>
      <width>61</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="textFormat">
     <enum>Qt::RichText</enum>
    </property>
    <property name="pixmap">
     <pixmap>../img/search-3.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>-1</x>
      <y>0</y>
      <width>1461</width>
      <height>851</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="textFormat">
     <enum>Qt::RichText</enum>
    </property>
    <property name="pixmap">
     <pixmap>../img/Sfondo.jpeg</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <zorder>label_3</zorder>
   <zorder>frame_3</zorder>
   <zorder>frame</zorder>
   <zorder>movementTable</zorder>
   <zorder>label_2</zorder>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
